packages/form-state-manager/src/utils/focus-error.ts

```diff
@@ -0,0 +1,17 @@
+import AnyObject from '../types/any-object';
+
+const focusError = (errors: AnyObject) => {
+  if (document) {
+    const allInputs = Array.from(document.forms)
```

Comment:
this might get messy when multiple forms are on. Can we pass the exact form element to the state manager? We could use ref for the react wrapper form component.

Reply:
I don’t think so, because the form element is not created by the manager but in the render function. So we had to create a ref and then pass it in the render function to the form - and it would tied this library to React. 

But we could easily use name attribute, but that would need users to set the name on form element and pass the same name to form manager if they run into this issue. What do you think?

Otherwise, it works the same as FF now. 😄

Reply:
Well, I don't think this is good. This can interfere with search inputs for instance. It's fairly common to have your search input wrapped in a form. Or any form that has input with the same name. I think we need some unique attribute (id or data) to distinguish between forms and not focus incorrect input.

Reply:
I am not sure what you mean.

You can put a name on form and the just find form with this name. It won’t interfere with any of inputs.

Reply:
Yeah, I meant the current solution, your suggestion would work.

TLDR, we need to uniquely specify the form instance to prevent potential focus conflicts between multiple forms. I believe you will come up with an adequate solution.